File: Solutions in Javascript/ExcelSheetColumnNumber.js
/**
 * @author : Anmol Agrawal
 */
/**
 * @param {string} s
 * @return {number}
 */
var titleToNumber = function(s) {
    let e = 1;
    let ans = 0;
    for(let i = s.length - 1; i >= 0; i--){
        ans += e * (s.charCodeAt(i) - 'A'.charCodeAt(0) + 1);
        e *= 26;
    }
    return ans;
};

File: Solutions in Java/ExcelSheetColumnNumber.java
/**
 * @author : Anmol Agrawal
 */
 class Solution {
    public int titleToNumber(String s) {
        int e = 1;
        int ans = 0;
        for(int i = s.length() - 1; i >= 0; i--){
            ans += (e * (s.charAt(i) - 'A' + 1));
            e *= 26;
        }
        return ans;
    }
}
